### DISCUSSION TITLE
How to store Exif data with a jpeg ?

### DISCUSSION THREAD

**Discussion by Zeus64** [ImageMagick/ImageMagick#3527] [Help]
Using magickWand, how to store exif data (the CreateDate) ? 
I tried to do 

`MagickSetImageProperty(Wand, 'exif:CreateDate', '2012:07:28 18:35:05') `

but when I save the jpg their is no exif info in the image :( What did I miss ?

**Comment by fmw42** [comment]
Imagemagick cannot write EXIF data, it can only read it.

**Reply by Zeus64** [reply]
Thanks :) however ImageMagick can remove the EXIF, I hope writing EXIF will be a feature included in futur release of Imagemagick, If we need to vote for this I vote :)
